.github/workflows/ci.yml

```diff
@@ -4,13 +4,26 @@ on:
     branches:
       - master
       - main
+permissions:
+  contents: write
 jobs:
   deploy:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v4
+      - name: Configure Git Credentials
+        run: |
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
```

Comment:
This is a hardcoded email address? How come?

Reply:
Yes this is recommended by mkdocs, so that the commits the bot makes are attributed correctly to the bot

https://github.com/squidfunk/mkdocs-material/pull/6335